src/main/java/com/online/bookstore/annotation/AdminAccessLevel.java

```diff
@@ -0,0 +1,12 @@
+package com.online.bookstore.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import org.springframework.security.access.prepost.PreAuthorize;
+
+@Retention(java.lang.annotation.RetentionPolicy.RUNTIME)
+@Target({ElementType.METHOD, ElementType.TYPE})
+@PreAuthorize("hasRole('ADMIN')")
+public @interface AdminAccessLevel {
+}
```

Comment:
Why do we need to create annotation in order to use it instead of other annotation?

Reply:
it looks better that just @PreAuthorize("hasRole('ADMIN')")

Reply:
can we proceed with this flyway versions? I'll create the new one in case db structure change